release-notes/6.0/README.md

```diff
@@ -1,10 +1,16 @@
-# .NET 6.0 Release
+# .NET 6
 
-Placeholder for .NET 6 release information.
+[.NET 6](https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-2/) is an [LTS release](../../release-policies.md) and will be supported for three years, from November 2021 to November 2024. It is [supported by Microsoft](../../microsoft-support.md) on [multiple operating systems](supported-os.md).
```

Comment:
I have mixed feelings about the verbiage here.  I wonder if talking about 6.0 as though it has been released is a bit confusing will be confusing for noobs.

Reply:
Yes. I added a statement above. It's the same approach that I'd use for EOL. Does that work?

Reply:
Yes, I like that callout.